3rdparty/libcxxrt/libcxxrt/CMakeLists.txt

```diff
@@ -1,13 +1,10 @@
 cmake_minimum_required(VERSION 3.0)
 
-enable_testing()
```

Comment:
Why remove this, and do other occurrences need to be removed?

Reply:
Some observations for this specific folder

https://github.com/openenclave/openenclave/blob/master/3rdparty/libcxxrt/libcxxrt/CMakeLists.txt is not used. Because this folder (`3rdparty/libcxxrt/libcxxrt`) is not added via `add_subdirectory()`

We can see no reference to this folder in its parent folder.
https://github.com/openenclave/openenclave/blob/master/3rdparty/libcxxrt/CMakeLists.txt

So the content in this file doesn't matter. Need to check if this is epected

Reply:
`enable_testing()` shouldn't be necessary if we use `include(CTest)` and set `BUILD_TESTING` to `ON`. CMake will automatically calls `enable_testing()` in that case.

Reply:
Excellent, thanks for the explanation!